code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -122,6 +122,20 @@
 			qdel(src)
 			user.put_in_hands(trash)
 
+/obj/item/storage/box/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/stack/packageWrap))
+		var/total_storage_space = W.get_storage_cost()
+		for(var/obj/item/I in contents)
+			total_storage_space += I.get_storage_cost()
+		if(total_storage_space <= max_storage_space)
+			var/question = alert(user, "Will you want to wrap \the [src] or store the item inside?", "Wrap or Store", "Wrap", "Store")
+			if(question == "Wrap")
+				return
+			else if(question == "Store")
+				return ..()
```

Comment:
Using click and drag to wrap stuff would be better than a selection box. (Alt click is already used for opening and closing fancy boxes.)

Imagine being a cargo tech and having to wrap multiple boxes. 
You'd have to click...go to the middle of the screen, click again, click on the box, go to the middle of the screen, click again...

Reply:
radial menu would also work here

Reply:
I already wrote ``desc_info`` for both the Wrapping Paper and Package Wrap, and the selection box only appears if the box has enough space for the package wrap to be stored in. If the box doesn't have the space for the package wrap, the package wrap automatically wraps it up and stuff

Reply:
> selection box only appears if the box has enough space for the package wrap to be stored in.

Still, there would be the issue of having to laboriously click over and over again if you are trying to wrap multiple things with space to fit the package wrap in. For example, a cargo tech could be wrapping many things, not only boxes, like backpacks and etc. and the popup issue will still occur.

Reply:
This only happens for the box in specific. It's... it's literally only for ``/obj/item/storage/box/``

Reply:
While I do think a radial menu looks better, I feel that the popup is not egregious. It is a rare instance, as stated. There is a desc_info to show how to use, so that's fine. I guess it's up to devs in the end, but I do not think a popup is bad in this case.